### ISSUE TITLE
No accuracy or test_accuracy reported

### ISSUE THREAD

**Issue by Taikakim** [maxfrenzel/SampleVAE#18]
I have now trained a few models, but I'm wondering if during training the parameters "accuracy" and "test_accuracy" should be reported? Now they don't have a value even at the end of training, example here:
```
"tep 3394; epoch 2.08; lr 0.000040 - loss = 2755.460, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.551 sec/step)
step 3395; epoch 2.08; lr 0.000040 - loss = 2531.649, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.532 sec/step)
step 3396; epoch 2.08; lr 0.000040 - loss = 2400.633, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.540 sec/step)
step 3397; epoch 2.08; lr 0.000040 - loss = 2308.458, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.560 sec/step)
step 3398; epoch 2.08; lr 0.000040 - loss = 2828.625, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.537 sec/step)
step 3399; epoch 2.08; lr 0.000040 - loss = 2244.958, accuracy = [], test_loss = 2349.611, test_accuracy = [], (1.548 sec/step)
Evaluating test set.
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 182/182 [01:50<00:00,  1.67it/s]
step 3400; epoch 2.08; lr 0.000040 - loss = 2288.676, accuracy = [], test_loss = 2403.894, test_accuracy = [], (111.571 sec/step)
No improvement on validation data for 5 test steps.                                                         Decreasing learning rate by factor 5.0
Reached learning rate threshold of 1e-05. Stopping training.
Storing checkpoint to model-AVP ...WARNING:tensorflow:Issue encountered when serializing trainable_variables.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'b_z0' has type str, but expected one of: int, long, bool
WARNING:tensorflow:Issue encountered when serializing variables.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'b_z0' has type str, but expected one of: int, long, bool
 Done.
kim@kim-HP-EliteDesk-800-G2-DM-65W:~/Downloads/SampleVAE-master$ `"`
```
